README.md

```diff
@@ -1,14 +1,123 @@
 
 # Customs Financials API
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90-green.svg)](/target/scala-3.3.5/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
 An API service for the CDS Financials project, which serves to:
 
-* proxy requests from the front-end (public) service, with circuit breaker protection, to HoDs (SPS, ETMP, etc.) 
+* proxy requests from the front-end (public) service, with circuit breaker protection, to HoDs (SPS, ETMP, etc.)
 
 This application lives in the "protected" zone. It integrates with:
 * Secure Payments Service (SPS) via Messaging Delivery Group (MDG)
 * Enterprise Tax Management Platform (ETMP) via MDG
 
+## Running the service
+
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9878
+
+### Required dependencies
+
+There are a number of dependencies required to run the service.
+
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+## Running the application locally
+
+| Command                              | Description                                                      |
+|--------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL` | Runs all dependencies                                            |
+| `sm2 -s`                             | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_API`  | Stop the micro service                                           |
+| `sbt run` or `sbt run 9878`          | (from root dir) to compile the current service with your changes |
+| `sbt "start -Dhttp.port=9878" `      | To run in `PROD` mode                                            |
+
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
+
+* `AUTH`
+* `AUTH_LOGIN_STUB`
+* `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
+* `USER_DETAILS`
+* `CUSTOMS_FINANCIALS_SDES_STUB`
+* `CUSTOM_DATA_STORE`
+* `CUSTOMS_FINANCIALS_HODS_STUB`
+* `CUSTOMS_FINANCIALS_EMAIL_THROTTLER`
+
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
```

Comment:
Better to rephrase this line as below as its a backend service. Remove line 60 as well

The service's endpoints (that need Enrolment to access) can be accessed by using below enrolments